gotham/Cargo.toml

```diff
@@ -25,7 +25,7 @@ hyper = "0.13.1"
 serde = "1.0"
 serde_derive = "1.0"
 bincode = "1.0"
-mime = "0.3"
+mime = "0.3.15"
```

Comment:
I did this because we are using `Mime::essence_str()` which is not available in `<3.15`